lib/src/main/kotlin/com/swmansion/starknet/provider/gateway/GatewayProvider.kt

```diff
@@ -82,7 +129,7 @@ class GatewayProvider(
         )
         val url = feederGatewayRequestUrl("get_storage_at")
 
-        return HttpRequest(Payload(url, "GET", params), Felt.serializer(), httpService)
+        return HttpRequest(Payload(url, "GET", params), buildHttpResponseDeserializer(Felt.serializer()), httpService)
```

Comment:
I think we could add a method here for creating a request with just the payload and the serialiser. It would wrap serializer with buildHttpResponseDeserializer and inject httpService.